scripts/resume.py

```diff
@@ -121,7 +121,7 @@ def instance_properties(nodeset, model, placement_group, labels=None):
 
         props.reservationAffinity = {
             "consumeReservationType": "SPECIFIC_RESERVATION",
-            "key": "compute.googleapis.com/reservation-name",
+            "key": f"compute.{universe_domain}/reservation-name",
```

Comment:
did you forget to call `universe_domain` ?

Reply:
Fixed.